### ISSUE TITLE
Create an asciinema recording to show how func-e works

### ISSUE THREAD

**Issue by peterj** [tetratelabs/func-e#381]
It would be nice to have a demo with [Asciinema](https://asciinema.org/) that shows how to install and run func-e.

**Comment by codefromthecrypt** [comment]
good idea! one nice to have on this would be that it could be re-used both in our README and the static site (that eventually becomes https://func-e.io). Currently, we just have static text for running func-e.
